internal/net/grpc/logger/logger_test.go

```diff
@@ -29,41 +30,48 @@ func TestInit(t *testing.T) {
 		name       string
 		want       want
 		checkFunc  func(want) error
-		beforeFunc func()
+		beforeFunc func(*testing.T)
 		afterFunc  func()
 	}
 	defaultCheckFunc := func(w want) error {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "set logger success with verbosity level is not set",
+			checkFunc: func(w want) error {
+				if grpclog.V(1) {
+					return errors.New("verbosity level is set")
+				}
+				return nil
+			},
+		},
+		// {
+		// 	name: "set logger success with verbosity level is set",
+		// 	beforeFunc: func(t *testing.T) {
+		// 		t.Setenv("GRPC_GO_LOG_VERBOSITY_LEVEL", "2")
+		// 	},
+		// 	checkFunc: func(w want) error {
+		// 		if !grpclog.V(1) {
+		// 			return errors.New("verbosity level 1 is not set")
+		// 		}
+		// 		if !grpclog.V(2) {
+		// 			//	return errors.New("verbosity level is not set")
+		// 		}
+		// 		if grpclog.V(3) {
+		// 			return errors.New("verbosity level is not correct")
+		// 		}
+		// 		return nil
+		// 	},
+		// },
```

Comment:
WIP?

Reply:
this test is ignored due to `Init()`  is executed automatically before the test is run. 
Should I remove this comment or should I write some comment about it?

Reply:
umm, we should consider how to run the `set log level` case.

Reply:
how about avoiding using inner function in `init()` and create a separate function to make it testable?

Reply:
can we separate the test functions?
e.g. `TestInit` -> test with no set value, `TestInit_SetEnv` -> test with setting env value.

if we could like it, we do not have to change the current implementation.

Reply:
We can't do it. 
The reason is that the `init()` is called when we import the package, the `once` object is initialized when we import it.
So the call from the second time (by the test code) will not work as the `once` object is initialized and the inner function will not executed at the second time.

Reply:
I see.
I wonder why you added this case, but we do not have to run.
If you'd like to add this case, could you please add any comments at the top of this case?

Reply:
```suggestion
		// this test case cannot be run because the once object is initialized
		// as the init() will be called when we import the package
		// {
		// 	name: "set logger success with verbosity level is set",
		// 	beforeFunc: func(t *testing.T) {
		// 		t.Setenv("GRPC_GO_LOG_VERBOSITY_LEVEL", "2")
		// 	},
		// 	checkFunc: func(w want) error {
		// 		if !grpclog.V(1) {
		// 			return errors.New("verbosity level 1 is not set")
		// 		}
		// 		if !grpclog.V(2) {
		// 			//	return errors.New("verbosity level is not set")
		// 		}
		// 		if grpclog.V(3) {
		// 			return errors.New("verbosity level is not correct")
		// 		}
		// 		return nil
		// 	},
		// },
```